backend/tests/Designer.Tests/Controllers/DeploymentsControllerTests.cs

```diff
@@ -56,7 +53,7 @@ public async Task GetDeployments_NoLaggingDeployments_PipelineServiceNotCalled()
                 .Setup(rs => rs.GetAsync(_org, _app, It.IsAny<DocumentQueryModel>()))
                 .ReturnsAsync(new SearchResults<DeploymentEntity> { Results = completedDeployments });
 
-            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, uri);
+            HttpRequestMessage httpRequestMessage = new(HttpMethod.Get, uri);
```

Comment:
Test classes, but the fix for this warning should be
```suggestion
            using var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, uri);
```